app/assets/javascripts/bookmarks/rename.js

```diff
@@ -1,17 +1,22 @@
-const originalValues = {};
+const originalNameTextboxes = {};
```

Comment:
I'm sure there's a better way to do these multiple JavaScript forms (maybe rails-ujs and data-remote?) but it seems like a straightforward extension of the current approach so I don't think it's a blocker!